src/python/pants/backend/python/providers/python_build_standalone/rules.py

```diff
@@ -204,25 +208,12 @@ async def get_python(
         ),
     )
 
-    await Get(
```

Comment:
hmm, so this is turns the operations from sync to async, which is generally not what one wants.

But poking around I'm not seeing an obvious handy async copytree like function to use.

ping maybe @benjyw since from the slack conversation.

Reply:
Shouldn't the execution of `shutil.copytree` here just be synchronous?

Reply:
Sorry! I said it in reverse.  I mean that before it was async with the `await` and the straight `shutil.copytree` is synchronous.

Reply:
If this is the desired behavior, I could create a util that submits a `shutil.copytree` task to a threadpool in an asynchronous fashion, but this should be a relatively fast operation so would love to understand the assumptions made here / requirements

Reply:
I think blocking on a synchronous `shutil.copytree` is probably fine at least to start.

Reply:
Agreed that blocking IO is good enough for now to unblock this working better, given we're likely to have to fundamentally reconsider the approach of seeding a local cache for these providers (https://github.com/pantsbuild/pants/issues/21439).

If we're having to make further changes to this PR anyway, maybe we could add a note to the comment (e.g. "(note: this is blocking IO, good enough for now, and the whole approach may need to be reconsidered with #21439 anyway).

(If no other changes required, I wouldn't bother updating the comment.)